### DISCUSSION TITLE
How to change language?

### DISCUSSION THREAD

**Discussion by pigeonskiller** [minios-linux/minios-live#7] [Q&A] [answered]
Hi, I'm a Linux enthusiast and distro hopper for pleasure.
I tested your distro and I must say that I found it interesting and very fast even on very old PCs.
Only one question arises for now: how to change language since there is no a language selector tool?

Thank you and have a nice day.

**Comment by crim50n** [comment, marked as answer]
> Hi, I'm a Linux enthusiast and distro hopper for pleasure. I tested your distro and I must say that I found it interesting and very fast even on very old PCs. Only one question arises for now: how to change language since there is no a language selector tool?
> 
> Thank you and have a nice day.

The distribution is small precisely for the reason that it has only one language. That is why there is a separate iso for each language.
If there are a large enough number of users who speak your language, I can add support for it, like I did for Spanish.
